pom.xml

```diff
@@ -899,8 +899,12 @@
                         <exclude>${test.files.integration}</exclude>
                     </excludes>
                     <runOrder>random</runOrder>
-                    <forkCount>1C</forkCount>
-                    <threadCount>1</threadCount>
+                    <!-- parallelize -->
+                    <parallel>classesAndMethods</parallel>
+                    <useUnlimitedThreads>true</useUnlimitedThreads>
+                    <threadCountClasses>7</threadCountClasses>
+                    <threadCountMethods>3</threadCountMethods>
```

Comment:
How we came up with this number?  If lets say a gitlab runner is not a multi-core does this count means more waiting for method execution?